src/components/Activities/Activities.jsx

```diff
@@ -4,114 +4,164 @@ import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
 import DateTime from 'react-datetime';
 import moment from 'moment';
 import {
-  getAllActivities,
-  handleEditActivities,
-  handleDeleteActivities,
-  handleInsertActivities,
+	getAllActivities,
+	handleEditActivities,
+	handleDeleteActivities,
+	handleInsertActivities
 } from './Activities_utils';
 
 class Activities extends Component {
-  constructor(props) {
-    super(props);
-    this.setTableData = this.setTableData.bind(this);
-    this.onAddRow = this.onAddRow.bind(this);
-    this.onDeleteRow = this.onDeleteRow.bind(this);
-    this.beforeSaveCell = this.beforeSaveCell.bind(this);
-
-    this.state = {
-      tableData: []
-    };
-  }
-
-  componentDidMount() {
-    let activitiesPromise = getAllActivities();
-    activitiesPromise.then(response => {
-      if (response.data) {
-        this.setTableData(response.data);
-      }
+	constructor(props) {
+		super(props);
+		this.setTableData = this.setTableData.bind(this);
+		this.onAddRow = this.onAddRow.bind(this);
+		this.onDeleteRow = this.onDeleteRow.bind(this);
+		this.beforeSaveCell = this.beforeSaveCell.bind(this);
+
+		this.state = {
+			tableData: []
+		};
+	}
+
+	componentDidMount() {
+		//e.preventDefault()
+		var activitiesPromise = getAllActivities();
+		activitiesPromise.then(response => {
```

Comment:
Do we really need to put this in a variable? Can't we just `.then` the promise right after calling `getAllActivities`?